api/v1alpha1/openapi.yaml

```diff
@@ -3650,6 +3669,29 @@ components:
           description: Reference to the container image for the application package.
       required:
         - image
+    InlineApplicationProvider:
```

Comment:
A small issue regarding consistency naming:


For configs, all the provider are named `TypeConfigProviderSpec`:
- GitConfigProviderSpec
- HttpConfigProviderSpec
- InlineConfigProviderSpec
- KubernetesSecretProviderSpec

Then it looks like we didn't name `ImageApplicationProvider` following that convention. That makes the new provider consistent with the existing Application provider, but the application providers are not consistent with the config providers.

Reply:
resolved